### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by YingtianDt** [NUST-Machine-Intelligence-Laboratory/VideoMAC#2]
Hi, I wonder if you can provide the pretrained weights. Thanks!

**Comment by Video-MAC** [comment]
Please see [VideoMAC Releases](https://github.com/Video-MAC/VideoMAC/releases).

**Comment by YingtianDt** [comment]
Thank you so much. In the releases I found the following checkpoints:
```
mac-tiny-r0.75.pth
mac-small-r0.75.pth
mac-r50-r0.75.pth
mac-r18-r0.75.pth
```

Could you also clarify on which dataset they were pretrained? And also the meaning of 'r0.75'? Thanks!

**Comment by Video-MAC** [comment]
pre-training data: YouTube-VOS 2018
r0.75: masking ratio

Please see our [arXiv paper](https://arxiv.org/abs/2402.19082) in Section 3.3 (pre-training) for more details.

**Comment by zouty-123** [comment]
> Please see [VideoMAC Releases](https://github.com/Video-MAC/VideoMAC/releases).

Hi! I would like to inquire about the differences between the VideoMAC and VideoMAC Releases codes.
